component/webui/src/main/java/org/exoplatform/social/webui/URLUtils.java

```diff
@@ -53,6 +59,11 @@ public static String getCurrentUser() {
     }
     
     String currentUserName = route.localArgs.get("streamOwnerId");
+    org.exoplatform.social.core.identity.model.Identity viewerIdentity = Utils.getViewerIdentity();
```

Comment:
after merging https://github.com/Meeds-io/social/pull/457/files Util.isExternal to be called instead